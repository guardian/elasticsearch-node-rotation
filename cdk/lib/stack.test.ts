import { App } from 'aws-cdk-lib';
import { Template } from 'aws-cdk-lib/assertions';
import { MyStack } from './stack';

describe('The IntegrationTest stack', () => {
	it('matches the snapshot', () => {
		const app = new App();
		const stack = new MyStack(app, 'IntegrationTest', {
			stack: 'cdk',
			stage: 'TEST',
		});
		const template = Template.fromStack(stack);
		expect(template.toJSON()).toMatchSnapshot();
	});
});
