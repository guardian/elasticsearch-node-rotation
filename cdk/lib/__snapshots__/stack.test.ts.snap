// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The IntegrationTest stack matches the snapshot 1`] = `Object {}`;
